src/EditorFeatures/Core/Shared/Utilities/ClassificationTypeMap.cs

```diff
@@ -3,61 +3,20 @@
 // See the LICENSE file in the project root for more information.
 
 using System;
-using System.Collections.Generic;
 using System.ComponentModel.Composition;
-using Microsoft.CodeAnalysis.Classification;
-using Microsoft.CodeAnalysis.ErrorReporting;
 using Microsoft.CodeAnalysis.Host.Mef;
 using Microsoft.VisualStudio.Text.Classification;
-using Roslyn.Utilities;
 
 namespace Microsoft.CodeAnalysis.Editor.Shared.Utilities
 {
     [Export]
-    internal class ClassificationTypeMap
+    internal class ClassificationTypeMap : AbstractClassificationTypeMap
```

Comment:
Ideally I think this class would be renamed to something like `DefaultClassificationTypeMap`, but I believe TypeScript takes a dependency on it so I don't think it can be changed in this PR without additional breaking changes.